index.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-process-exit */
+const { program } = require("commander");
+const loadFondue = require("fondue");
+const packageJson = require("./package.json");
+const { stdout } = require("process");
+
+program
+	.name("wakamai-fondue")
+	.version(packageJson.version)
+	.arguments("<file>")
+	.action(run);
+
+program.option("-c, --css", "Generate CSS");
+program.option("-i, --info", "Dump info about font");
```

Comment:
I'm not too sure about the name. Perhaps this should be called `--json`. As in: `What can my font do? Tell me in JSON please.`.

Reply:
Good suggestion, with `--info` I would assume some basic font info, not "everything". Go for it! 👍

Reply:
Renamed to `--json`.